swing/src/main/java/info/openrocket/swing/gui/simulation/SimulationConditionsPanel.java

```diff
@@ -166,11 +167,38 @@ public static void addSimulationConditionsPanel(JPanel parent, SimulationOptions
 		sub.add(slider, "w 75lp, wrap");
 
 
+		// Humidity:
+		label = new JLabel(trans.get("simedtdlg.lbl.Humidity"));
+		////The humidity at the launch site.
+		tip = trans.get("simedtdlg.lbl.ttip.Humidity");
+		label.setToolTipText(tip);
+		isa.addEnableComponent(label, false);
+		sub.add(label);
+
+		humidityModel = new DoubleModel(target, "LaunchHumidity", UnitGroup.UNITS_NONE, 0);
```

Comment:
Please us `UnitGroup.UNITS_RELATIVE`, since relative humidity is expressed as a percentage.